script/Recall.s.sol

```diff
@@ -42,8 +42,8 @@ contract DeployScript is Script {
         ) {
             console.log("Deploying token manager");
             IInterchainTokenService itsContract = IInterchainTokenService(INTERCHAIN_TOKEN_SERVICE);
-            bytes memory params = abi.encode(abi.encodePacked(recall.deployer()), address(recall));
-            itsContract.deployTokenManager(itsSalt, "", ITokenManagerType.TokenManagerType.MINT_BURN_FROM, params, 0);
+            bytes memory linkParams = abi.encodePacked(recall.deployer());
+            itsContract.linkToken(itsSalt, network, address(recall), ITokenManagerType.TokenManagerType.MINT_BURN_FROM, linkParams, 0);
```

Comment:
With the new `linkToken` feature, token linking should be initiated from a fixed source chain. You can pick a major chain that your token already exists on, and provide the chain you're registering the new token on as the destinationChain argument (`network`) above.

`linkParams` is now just the address of the optional operator being set

Once `linkToken`, it'll create a cross-chain message to deploy the token manager on the remote chain. the token manager address will be the same, and the tokenId will be the same